AsyncDisplayKit/ASMapNode.mm

```diff
@@ -41,11 +40,9 @@ - (instancetype)init
   _liveMap = NO;
   _centerCoordinateOfMap = kCLLocationCoordinate2DInvalid;
   
-  //Default world-scale view
-  _region = MKCoordinateRegionForMapRect(MKMapRectWorld);
-  
   _options = [[MKMapSnapshotOptions alloc] init];
-  _options.region = _region;
+  //Default world-scale view
+  _options.region = MKCoordinateRegionForMapRect(MKMapRectWorld);
```

Comment:
I totally agree. I'll move this to `-setUpSnapshotter` so that it is only allocated when needed. For the bigger picture of avoid creating objects in -init, I think that is a very worthwhile strategy to undertake, since often we init a node before we use it, so this will help :)